plugins/git/README.md

```diff
@@ -118,6 +123,7 @@ plugins=(... git)
 | gpv                  | git push -v                                                                                                                      |
 | gr                   | git remote                                                                                                                       |
 | gra                  | git remote add                                                                                                                   |
+| grao                 | git remote add origin                                                                                                            |
```

Comment:
It might be best to remove this, as per [here](https://github.com/ohmyzsh/ohmyzsh/pull/8932#issuecomment-640552401)